web/src/components/CircleIcon.tsx

```diff
@@ -0,0 +1,58 @@
+import interview from "~public/icons/interview.svg";
+import check from "~public/icons/check.svg";
+import list from "~public/icons/list.svg";
+import {useState} from "react";
+import Image from 'next/image';
+
+interface CircleIconProps {
+  icon: string;
+  message: string;
+}
+
+export const CircleIcon = ({icon, message}: CircleIconProps) => {
+  const [isHovered, setIsHovered] = useState(false);
+
+  return (
+    <div
+      className="relative flex items-center justify-center"
+      onMouseEnter={() => setIsHovered(true)}
+      onMouseLeave={() => setIsHovered(false)}
+    >
+      <div className={`rounded-full h-20 w-20 md:h-28 md:w-28 flex items-center justify-center transition-colors duration-100 ${
+        isHovered ? 'bg-red-700' : 'bg-gray-600'
+      }`}>
+        {icon === 'list' && (<Image
+          src={list}
+          style={{filter: 'invert(1)'}}
+          className="w-8/12 md:w-8/12"
+          alt={icon}
+        />)}
+        {icon === 'interview' && (<Image
+          src={interview}
+          style={{filter: 'invert(1)'}}
+          className="w-9/12 md:w-9/12"
+          alt={icon}
+        />)}
+        {icon === 'check' && (<Image
+          src={check}
+          style={{filter: 'invert(1)'}}
+          className="w-9/12 md:w-7/12"
+          alt={icon}
+        />)}
```

Comment:
We could put the variable props for the different variations in an object to remove some repetition.
Something like:
```js
  const variants = {
    list: {
      src: list,
      className: 'w-8/12 md:w-8/12'
    },
   interview: {...}
  }

  (
    <Image
        style={{filter: 'invert(1)'}}
        alt={icon}
        {...variants[icon]}
        />
  )
```
What do you think?

Reply:
cool